src/main/java/com/kunmi/taskManager/utils/redis/RedisUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.kunmi.taskManager.utils.redis;
+
+import redis.clients.jedis.Jedis;
+
+public class RedisUtil {
+
+    private static final String REDIS_HOST = "localhost";
+    private static final int REDIS_PORT = 6379;
+
+    public static Jedis getJedis() {
+        return new Jedis(REDIS_HOST, REDIS_PORT);
```

Comment:
I think this line create new connection every time when you call this method, you should aviod this